### PR TITLE
Lodge Rework Part 0.1

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request
Suggestion is:

I want to give hunters the following perks:

Anomaly Hunter
For all of Lodge's roles due to their necessity of being outside of the colony and only relaying on themselfs to know what oddity do what (Also lore-wise its explained they got teachings from folkens that know how anomalies works)

For Lodge Hunt Master and Hunter
Green Thumb
Due to well... messing with plants

And add a new Role to lodge.

### CODE REVIEW

**Comment by TheSleepyCatmom** [comment]
My suggestion was just to add a new role and not to give the hunters different skills to prevent making them all capable of everything, just to remind. :bap:

**Comment by Guidesu** [comment]
> My suggestion was just to add a new role and not to give the hunters different skills to prevent making them all capable of everything, just to remind. :bap:

oh alright

**Comment by cdb-is-not-good** [comment]
Giving them anomaly hunter is a hard no. Giving herbalist green thumb, however; that's fine

**Comment by Spoopy11** [comment]
If Hunters get anomaly hunter, Scavs get scav step. :^)

File: code/game/jobs/job/offcolony.dm [hunk 1]
```diff
@@ -6,15 +6,15 @@
...
+	supervisors = "the Lodge Matriarch"
 	difficulty = "Hard."
 	selection_color = "#9b633e"
 	account_allowed = 0
 	create_record = 0
 	wage = WAGE_NONE
 	outfit_type = /decl/hierarchy/outfit/job/off_colony/hunt_master
 
-	perks = list(/datum/perk/job/butcher)
+	perks = list(/datum/perk/job/butcher, /datum/perk/stalker, /datum/perk/job/master_herbalist, /datum/perk/greenthumb)
```
Comment (cdb-is-not-good) [inline code/game/jobs/job/offcolony.dm:17]: No.....why?

File: code/game/jobs/job/offcolony.dm [hunk 2]
```diff
@@ -48,14 +48,14 @@
...
-	supervisors = "the hunt master"
+	supervisors = "the Hunt Master"
 	selection_color = "#9b633e"
 	account_allowed = 0
 	create_record = 0
 	wage = WAGE_NONE
 	outfit_type = /decl/hierarchy/outfit/job/off_colony/hunter
 
-	perks = list(/datum/perk/job/butcher)
+	perks = list(/datum/perk/job/butcher, /datum/perk/stalker,  /datum/perk/greenthumb)
```
Comment (cdb-is-not-good) [inline code/game/jobs/job/offcolony.dm:58]: Again, why?

File: code/game/jobs/job/offcolony.dm [hunk 3]
```diff
@@ -119,6 +119,46 @@
 	icon_state = "player-black"
 	join_tag = /datum/job/off_colony_herbalist
 
+/datum/job/off_colony_technician
```
Comment (cdb-is-not-good) [inline code/game/jobs/job/offcolony.dm:122]: A whole new job for this seems...unnecessary. It also goes against their general 'thing' as outdoorsmen

**Comment by ThePainkiller** [comment]
This makes:

- Hunters literally Prospector 2 + gardener perk
- Creates Scavenger2 but with access to every lodge perk for the sake of crafting stuff (and being literally a stone throw from scrap heaven to make immediate use of that perk)
- Conflicts with Scavengers claims to most dungeons Lodge would get their mats from

I'm not against giving Hunt Master the green thumb perk either but the rest is...why?

**Comment by Guidesu** [comment]
> This makes:
> 
>     * Hunters literally Prospector 2 + gardener perk
> 
>     * Creates Scavenger2 but with access to every lodge perk for the sake of crafting stuff (and being literally a stone throw from scrap heaven to make immediate use of that perk)
> 
>     * Conflicts with Scavengers claims to most dungeons Lodge would get their mats from
> 
> 
> I'm not against giving Hunt Master the green thumb perk either but the rest is...why?

i dunno it sounded funny

File: code/game/jobs/job/offcolony.dm (hunk 3 above)
Comment (Guidesu) [reply code/game/jobs/job/offcolony.dm:122]: I dunno. Mitten said to create one instead of giving hunters a more general skillset

Comment (cdb-is-not-good) [reply code/game/jobs/job/offcolony.dm:122]: I suppose theses are two equal solutions to a problem that I'm...not sure Hunters have.

**Comment by LongSleeves** [comment]
Not really horrible. Scavs and Hunters mostly have different looting locations and salvaging rights. Not sure if hunters have actual looting rights at scrap haven or not. If they do then their might be conflict, but beyond that. I could see building being useful for when hunters head to swamp to do things. Ideally we start having someone crack down if hunters start heading to prospie areas to loot sense its a break of the codex last I checked. A roundabout way for hunters to get to swamp thru the plains river could help to keep them out of the prospie path in jungle too.

**Comment by Guidesu** [comment]
> Going to be giving a big ol' 'why?' to this Pr- the lodge neither needs a new job nor do they need to all have these perks.

Actually i just wanted to give the herbalist 20+ mechanical because time to time they need to heal lodge's synths.
But RadiattenMitten said to me to create an entire un-related role that even i is confused why its exist in the first place

**Comment by Guidesu** [comment]
Taking a break from sojourn

**Pull request closed without merging**